website_country_localized_pages/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Country specific web pages
+==========================
+
+Use this module to create localized web pages according to visitor's country.
+
+
+Installation
+============
+
+To install this module, you need ipwhois
+https://pypi.python.org/pypi/ipwhois
+
+
+Configuration
+=============
+
+Through CMS (frontend website) interface, create for instance a new 'US features' page for Unites States, without adding it to the menu, and fill it with US specific content.
+
+Then, go to
+Settings -> Technical -> User interface -> Views
+search for standard 'features' page, go to 'country specific views' tab, add one line with 'US' country and 'US features' page.
```

Comment:
@Yajo I replaced ```with``` with ```selecting``` because user has to select the country from the list (```country_id``` is ```Many2one('res.country', ...```). So, country codes are already available in the system

About example generalization, see https://github.com/OCA/website/pull/59#issuecomment-130564858